<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterContractsForm</class>
 <widget class="QWidget" name="RegisterContractsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>94</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Договор</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/main.qrc">
    <normaloff>:/images/logo.png</normaloff>:/images/logo.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>Ок</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="gui::ContractInformationWidget" name="ContractInformationWidget" native="true"/>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="applyButton">
     <property name="text">
      <string>Применить</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="exportButton">
     <property name="text">
      <string>Экспорт</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="endContractButton">
     <property name="text">
      <string>Завершить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>gui::ContractInformationWidget</class>
   <extends>QWidget</extends>
   <header location="global">gui/widgets/contractinformationwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>okButton</tabstop>
  <tabstop>applyButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources/main.qrc"/>
 </resources>
 <connections/>
</ui>
